brainiak/matnormal/regression.py

```diff
@@ -0,0 +1,155 @@
+import tensorflow as tf
+import numpy as np
+from sklearn.base import BaseEstimator
+from brainiak.matnormal.matnormal_likelihoods import matnorm_logp
+from tensorflow.contrib.opt import ScipyOptimizerInterface
+
+__all__ = ['MatnormRegression']
+
+
+class MatnormRegression(BaseEstimator):
+    """ This analysis allows maximum likelihood estimation of regression models
+    in the presence of both spatial and temporal covariance.
+
+    ..math::
+    Y \\sim \\mathcal{MN}(X\beta, time_cov, space_cov)
+
+    Parameters
+    ----------
+    time_cov : subclass of CovBase
+        TR noise covariance class following CovBase interface.
+    space_cov : subclass of CovBase
+        Voxel noise covariance class following CovBase interface.
+    learnRate : real, default=0.01
```

Comment:
Apologies if I'm missing something, but input argument learnRate doesn't seem to be an argument in the __init__ below. Do we need documentation for optimizer method and optCtrl?

Reply:
yeah, outdated docs, will fix this.